presto-orc/src/main/java/com/facebook/presto/orc/OrcReader.java

```diff
@@ -222,12 +222,12 @@ public CompressionKind getCompressionKind()
         return compressionKind;
     }
 
-    public OrcRecordReader createRecordReader(Map<Integer, Type> includedColumns, OrcPredicate predicate, DateTimeZone hiveStorageTimeZone, AggregatedMemoryContext systemMemoryUsage, int initialBatchSize)
+    public OrcBatchRecordReader createRecordReader(Map<Integer, Type> includedColumns, OrcPredicate predicate, DateTimeZone hiveStorageTimeZone, AggregatedMemoryContext systemMemoryUsage, int initialBatchSize)
     {
         return createRecordReader(includedColumns, predicate, 0, orcDataSource.getSize(), hiveStorageTimeZone, systemMemoryUsage, initialBatchSize);
     }
 
-    public OrcRecordReader createRecordReader(
+    public OrcBatchRecordReader createRecordReader(
```

Comment:
Should this be called `createBatchRecordReader`?

Reply:
@tdcmeehan @highker I'm fine either way, but I feel that if we rename this method, then, perhaps, we should rename all of the following as well:

- OrcPageSource -> OrcBatchPageSource
- OrcPageSourceFactory -> OrcBatchPageSourceFactory
- DwrfPageSourceFactory -> DwrfBatchPageSourceFactory
- OrcReader -> OrcBatchReader

What do you think?

Reply:
If we end up with two different page sources, then probably yes.

Reply:
@highker Sounds good. Would you take a look at `Introduce SelectiveStreamReader` commit. I'll do these renames while addressing feedback on that.

Reply:
Yes, actually let me do a review before changing the names